spec/lib/blacklight/document/cache_key_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+
+describe Blacklight::Document::CacheKey do
+  let(:attributes) { {} }
+  let(:subject) { SolrDocument.new(attributes) }
+  it 'SolrDocument includes the module' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.